### PR TITLE
Va gov/content wip apply for adaptive housing

### PR DESCRIPTION
**Pull request by DanielleSoCompany**
## Description
Changing Adaptive Housing page url/title and adding a How to Apply sub page.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by DanielleSoCompany** [comment]
Hi @ncksllvn @RobAWilkinson @accbjt. The copy for this PR is ready to merge, but there are 2 things that need attention and I'm not sure how to fix so would appreciate your help:

- The left nav doesn't seem to be showing up on this page: va-gov/pages/housing-assistance/disability-housing-grants/how-to-apply.md
- On both https://vetsgov-pr-8390.herokuapp.com/housing-assistance/disability-housing-grants/ and va-gov/pages/housing-assistance/disability-housing-grants/how-to-apply.md, in the green "Go to eBenefits to Apply" button, the text seems to be too far over to the right (as below)

image

Thanks in advance for your help!

File: va-gov/pages/housing-assistance/disability-housing-grants.md [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
 layout: page-breadcrumbs.html
 template: detail-page
-title: Adaptive Housing Grants
-display_title: Housing Grants
-concurrence:
-order: 1
-spoke: Disability Housing Grants
-children: disabilityHousingGrants
+title: About Disability Housing Grants for Veterans
+display title: Housing Grants
```
Comment (accbjt) [inline va-gov/pages/housing-assistance/disability-housing-grants.md]: This needs a `_`

File: va-gov/pages/housing-assistance/disability-housing-grants/how-to-apply.md [hunk 2]
```diff
@@ -1,5 +1,97 @@
 ---
-title: How to Apply
-href: /housing-assistance/disability-housing-grants/how-to-apply/
+layout: page-breadcrumbs.html
+template: detail-page
+title: How to Apply for a Specially Adapted Housing (SAH) Grant
+display title: How to Apply
```
Comment (accbjt) [inline va-gov/pages/housing-assistance/disability-housing-grants/how-to-apply.md]: same

**Review by accbjt — CHANGES_REQUESTED** [review]
Looks good you just need to change `display title` to `display_title`.

**Review by accbjt — APPROVED** [review]

**Pull request merged**